test/function/samples/tree-shake-iterable/main.js

```diff
@@ -0,0 +1,29 @@
+let effects = 0;
+
+const iterable = {
+	[Symbol.iterator]() {
+		return {
+			next() {
+				effects++;
+				return { done: true };
+			}
+		};
+	}
+};
+
+new Map(iterable);
+new Set(iterable);
+new WeakMap(iterable);
+new WeakSet(iterable);
+Float32Array.from(iterable);
+Float64Array.from(iterable);
+Int16Array.from(iterable);
+Int32Array.from(iterable);
+Int8Array.from(iterable);
+Uint16Array.from(iterable);
+Uint32Array.from(iterable);
+Uint8Array.from(iterable);
+Uint8ClampedArray.from(iterable);
```

Comment:
we could add `Array.from`, `BigInt64Array.from`, `BigUint64Array.from` as well.

Reply:
Oh, I didn't take this into account. I have changed it now.